geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableRegistrationDUnitTest.java

```diff
@@ -345,6 +338,15 @@ public void run2() throws CacheException {
     durableClientVM
         .invoke(() -> DurableRegistrationDUnitTest.registerKey(K2, Boolean.FALSE));
 
+
+    // Step 9: Send clientReady message
+    durableClientVM.invoke(new CacheSerializableRunnable("Send clientReady") {
+      @Override
+      public void run2() throws CacheException {
+        CacheServerTestUtil.getCache().readyForEvents();
+      }
+    });
```

Comment:
See above comment about changing this to have fewer warnings.

Reply:
Done

Reply:
Done.